app/domain/authentication/security/validate_role_can_access_webservice.rb

```diff
@@ -18,13 +18,14 @@ module Security
     ) do
 
       def call
-        # No checks required for default conjur authn
+        validate_account_exists
+        validate_role_is_defined
```

Comment:
It looks like these caused some rspec failures that will require test updates.

Reply:
Done, thanks